docs/Reference/index.md

```diff
@@ -1,8 +1,71 @@
 <h1 align="center">Fastify</h1>
 
-## Reference Documentation Table Of Contents
+## Core Documents
+<a id="reference-core-docs"></a>
+
+For the full table of contents (TOC), see [below](#reference-toc). The following
+list is a subset of the full TOC that detail core Fastify APIs and concepts in
+order of most likely importance to the reader:
```

Comment:
Not 100% on this line, I don't know if "assumed importance" would read better?

Reply:
I'm going to go ahead and merge this. If your wife would like to tie break this one, or we can think of something better, let's PR a change.